understanding/22/consistent-help.html

```diff
@@ -43,49 +43,48 @@ <h2>Consistent Help</h2>
 
         <p>When a user is quickly able to find help, they are able to complete the task even if they encounter challenges.</p>
 
-        <p>At least one of the following mechanisms to get help should be included on all websites:</p>
-			<ul>
-            <li>Human contact details such as a phone number, email address, hours of operation.</li>
-            <li>Human contact mechanism such as a messaging system, chat client, contact form, social media channel.</li>
-            <li>Self-help option such as an up-to-date Frequently Asked Questions, How Do I page, Support page.</li>
-            <li>A fully automated contact mechanism such as a chatbot.</li>
-         </ul>
-         <p>While it would be best for all sites to offer at least one of these types of help, this SC does not require help be provided. The SC requires that when help is available that it be in a consistent location.</p>
-
-        <p>The human contact details should enable the user to connect with the  organization or the part of the organization that can assist with the content. For example, an online jobs / recruitment portal should provide a contact method for the team that supports the recruitment portal and not a catch-all for the entire company. Each layer of contact added prolongs the time before the user will receive help.</p>
-
-        <p>The human contact mechanism enables a person to express what they are looking for using their own words. For some with cognitive disabilities, this may be the best way for them to find an answer to their problem.</p>
-
-        <p>For pages for which no human support is available, a self-help option should say that no human support is available.  Self-help options should go beyond allowing the user to search within the site. Contextual help is still recommended (See SC 3.3.5 for more information) but a self-help option should provide a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
+        <p>Users should not need to resort to internet searches to find the contact information for an organization. Further, users struggling to complete a task on a site shouldn't encounter additional cognitive challenges when searching for help <em>within</em> the site. The user’s disability may make it more difficult to find the help available if the information is not present within a few interactions.</p>
+        <h3>Forms of help</h3>
```

Comment:
i'd couch this whole section more clearly into a "we're only giving some examples here, nothing here is normatively required by the SC" frame with some opening statement, as it otherwise still gives off the vibe that authors have to pick one in order to comply, which goes beyond the scope of the SC